android/app/src/production/google-services.json

```diff
@@ -0,0 +1,85 @@
+{
+    "project_info": {
+      "project_number": "897975858437",
+      "firebase_url": "https://analogio-47cf0.firebaseio.com",
+      "project_id": "analogio-47cf0",
+      "storage_bucket": "analogio-47cf0.appspot.com"
+    },
+    "client": [
+      {
+        "client_info": {
+          "mobilesdk_app_id": "1:897975858437:android:5496e6f1f212adf7e3336c",
+          "android_client_info": {
+            "package_name": "com.analogio.coffeecard"
+          }
+        },
+        "oauth_client": [
+          {
+            "client_id": "897975858437-en6jjspk6sn0d7qhathuih0kih4iv0nc.apps.googleusercontent.com",
+            "client_type": 3
+          }
+        ],
+        "api_key": [
+          {
+            "current_key": "AIzaSyBSOwHLlN9ig7f0fLvQ7nra5OfWgbetT4c"
+          }
+        ],
+        "services": {
+          "appinvite_service": {
+            "other_platform_oauth_client": [
+              {
+                "client_id": "897975858437-en6jjspk6sn0d7qhathuih0kih4iv0nc.apps.googleusercontent.com",
+                "client_type": 3
+              },
+              {
+                "client_id": "897975858437-5mtqt6dn8qarvrb8ebfujvvkb5nldcoi.apps.googleusercontent.com",
+                "client_type": 2,
+                "ios_info": {
+                  "bundle_id": "DK.AnalogIO.DigitalCoffeeCard",
```

Comment:
I think it's a bit frustrating that we don't have a specific "naming scheme" for our id's - sometimes the app is referred to as "DigitalCoffeeCard", other times "digitalclipcard", sometimes we are "analog", sometimes we are "analogio"

Can we do anything to "unify" it?

Reply:
The apps were originally two separate projects, hence the different names. The Flutter app just took over from the previous naming schemes so that people with the old apps could update to the Flutter app via an automatic App Store update.

If we want to change the app id, the only thing we can do is to sunset the current apps (take them off app stores) and re-release them under a new app id.

Reply:
Loosely related: because the old iOS app stated support for iPad, we currently have to support iPad as well. We haven't designed this app for iPad, so if anyone tries to download this app on an iPad, the result will be ugly.

Again, the only solution is to take the current app off the App Store. According to Apple, we can always add iPad support in the future, but once you have added a supported platform, you have to support it forever.

Reply:
Yes alright, I see 👍